test/gtest/Makefile.am

```diff
@@ -55,7 +57,8 @@ gtest_CPPFLAGS = \
 
 gtest_LDFLAGS  = $(GTEST_LDFLAGS) -no-install
 gtest_CFLAGS   = $(BASE_CFLAGS)
-gtest_CXXFLAGS = $(BASE_CXXFLAGS) $(GTEST_CXXFLAGS) -g -fno-tree-vectorize
+gtest_CXXFLAGS = $(BASE_CXXFLAGS) $(GTEST_CXXFLAGS) -g -fno-tree-vectorize \
+				 -DGTEST_UCM_HOOK_LIB_DIR="\"${PWD}/ucm/test_dlopen/.libs\""
```

Comment:
pls use automake vars (e.g abs_builddir) instead of PWD

Reply:
looks like it's not fixed..

Reply:
removed -g and missed about PWD :)
fixed